controller/src/beerocks/master/son_master_thread.cpp

```diff
@@ -607,8 +607,8 @@ bool master_thread::handle_cmdu_1905_autoconfiguration_WSC(Socket *sd,
     std::shared_ptr<wfa_map::tlvApRadioBasicCapabilities> radio_basic_caps = nullptr;
     std::shared_ptr<ieee1905_1::tlvWscM1> tlvwscM1                         = nullptr;
     bool intel_agent                                                       = false;
-    while (1) {
-        int type = cmdu_rx.getNextTlvType();
+    int type = cmdu_rx.getNextTlvType();
+    do {
```

Comment:
Do we want to avoid this pattern?
```
int type;
while ((type = cmdu_rx.getNextTlvType()) != int(ieee1905_1::eTlvType::TLV_END_OF_MESSAGE))
```

Reply:
we don't, personal taste. But since both you and Moran like this one better, I'll align :)